sdk/identity/azure-identity/azure/identity/_credentials/imds.py

```diff
@@ -29,7 +29,7 @@
 }
 
 
-def get_request(scope, identity_config):
+def get_request(scope: str, identity_config: Dict):
```

Comment:
is this public? what does it return?

Reply:
No it is not public, I renamed it to _get_request.